src/test/resources/cart.json

```diff
@@ -52,23 +52,39 @@
                 "amount": 10000
             },
             "taxInfo": {
-                "amount": 1000,
+                "value": {
+                    "currency": "USD",
+                    "amount": "900",
```

Comment:
Sorry I initially missed that, all these amounts should be numbers, not strings (there are quite a few in many JSON examples). BTW, you changed the amounts, while I initially made sure the amounts match so that the example makes sense.